api/v1alpha1/sysbench_types.go

```diff
@@ -21,8 +21,8 @@ import (
 )
 
 // SysbenchSpec contains the configuration parameters
-// with scheduling options for the sysbench benchmark.
-// The options, testName and command parameters are passed
+// with scheduling arguments for the sysbench benchmark.
```

Comment:
'Scheduling options' was referring to the PodSchedulingSpec. I don't think we should rename that

Reply:
reverted